src/controllers/queries/__tests__/helpers.ts

```diff
@@ -43,9 +43,13 @@ function templateListFake(templateName: string, ...args: any[]) {
 export const withQueriesMocks = (stage: Scope3FormStage = 'companySelect', compNumber: string = '10000009') => {
   const scope3CarbonConsumptionTemplateMock = {
     newScope3CarbonConsumptionFormPage: (
+      // @ts-ignore
```

Comment:
why? Can we not fix whatever the warnings were?

Reply:
unused variable

Reply:
if we don't use the variable can we just rmeove it?

Reply:
it has been removed and should be reflect in the file changes upon next push